### ISSUE TITLE
Can't pass rr-opts

### ISSUE THREAD

**Issue by NickHu** [danielzfranklin/cargo-rr#11]
No matter what I try to do, I can't pass options to `rr`. For example:

```
❯ cargo rr replay mytrace -s 1234 
error: Found argument '-s' which wasn't expected, or isn't valid in this context

USAGE:
    cargo rr replay [trace] [rr-opts] [-- <gdb-opts>...]

For more information try --help
```

**Comment by NickHu** [comment]
So the problem is essentially that any `rr` option you would want to use begins with `-`, which then gets intercepted by `clap` rather than passed through.

**Comment by dzfranklin** [comment]
Fixed in #14
